WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -689,6 +709,32 @@ public void onPostChanged(OnPostChanged event) {
     public void onPostUploaded(OnPostUploaded event) {
         if (isAdded() && event.post.getLocalSiteId() == mSite.getId()) {
             loadPosts(LoadMode.FORCED);
+
+            final PostModel post = event.post;
+            boolean isDraft = post != null && PostStatus.fromPost(post) == PostStatus.DRAFT;
+            if (isDraft) {
+                if (PostUtils.isPublishable(post)) {
```

Comment:
So I might be missing something else, but I think there's some confusion here because of the naming of `isPublishable`. A better name for it would probably have been `isUploadable`. If you send a post to WordPress without at least one of `content`, `title`, or `excerpt`, the upload will fail, no matter what status the post is supposed to have (`published`, `draft`, etc).

So if we receive an `OnPostUploaded` event without an error, we should know with certainty that the `isPublishable` check will be true (and if it's a remote draft, it can be published).